src/Microsoft.AspNetCore.Mvc.Razor.Host/MvcRazorTemplateEngine.cs

```diff
@@ -52,6 +52,8 @@ private static RazorSourceDocument GetDefaultImports()
                 writer.WriteLine("@inject global::Microsoft.AspNetCore.Mvc.IUrlHelper Url");
                 writer.WriteLine("@inject global::Microsoft.AspNetCore.Mvc.ViewFeatures.IModelExpressionProvider ModelExpressionProvider");
                 writer.WriteLine("@addTagHelper Microsoft.AspNetCore.Mvc.Razor.TagHelpers.UrlResolutionTagHelper, Microsoft.AspNetCore.Mvc.Razor");
+                writer.WriteLine("@addTagHelper Microsoft.AspNetCore.Mvc.Razor.TagHelpers.HeadTagHelper, Microsoft.AspNetCore.Mvc.Razor");
+                writer.WriteLine("@addTagHelper Microsoft.AspNetCore.Mvc.Razor.TagHelpers.BodyTagHelper, Microsoft.AspNetCore.Mvc.Razor");
```

Comment:
This code is in the process of moving to Razor. You should also add this at https://github.com/aspnet/Razor/blob/dev/src/Microsoft.AspNetCore.Mvc.Razor.Extensions/MvcRazorTemplateEngine.cs#L54